### ISSUE TITLE
AMD RX6750XT for macos13

### ISSUE THREAD

**Issue by MizhiDR** [ChefKissInc/NootRX#13]
when i check NootRX Can be displayed in recovery mode
But when I want to enter the system, I reboot.
**Screenshots**

**System Information:**
 - macOS Version: [e.g. macOS 13.1]
 - CPU: [e.g. Intel i5 12400F]
 - GPU: [e.g. AMD Radeon RX 6700 XT]

**Additional context**
![微信图片_20231116161705](https://github.com/ChefKissInc/NootRX/assets/28851784/09204861-b0a9-4cef-9c0a-50e034063d43)

**Comment by ExtremeXT** [comment]
@MizhiDR > macOS 13.1
This project only supports the latest subversion of each major macOS version, update to 13.6.2 or whatever.

**Comment by MizhiDR** [comment]
> @MizhiDR> macOS 13.1 此项目仅支持各个主要 macOS 版本的最新颠覆，更新到 13.6.2 或其他版本。

Doesn’t support 13.1?

**Comment by VisualEhrmanntraut** [comment]
@MizhiDR No. As was displayed before you submitted this issue, only latest minor (X.Y.Z, X is major and Y is minor) of supported major macOS versions is supported.

**Comment by MizhiDR** [comment]
> @MizhiDR否。正如您提交此问题之前所显示的，仅支持受支持的主要 macOS 版本的最新次要版本（XYZ、X 是主要版本、Y 是次要版本）。

Thank you, I'll try

**Comment by ExtremeXT** [comment]
@MizhiDR
Is this issue still relevant?

**Comment by fainc** [comment]
兄弟，有版本成功吗，我6750XT 14.1也是和你描述的一样安装和恢复模式正常，进系统卡死

**Comment by ExtremeXT** [comment]
@fainc Use English...

**Comment by holoto** [comment]
> 兄弟，有版本成功吗，我6750XT 14.1也是和你描述的一样安装和恢复模式正常，进系统卡死

me too. 
install macOS  version : Sonoma 14 (22A344) 
hardware spec : 12400+ rx6700xt
install all good work. but boot macos in local disk  
always error lag , wait 5min, that log code show 240s  gpu0

**Comment by holoto** [comment]
> @holoto Disable macOS until you're in Setup Asisstant then enable it again, Lilu bug.

I tested this method, disabling the NootRX driver until I entered the system desktop, and then re-enabled it, and now the problem is solved, thanks

**Comment by ExtremeXT** [comment]
Closing.

**Comment by scanduz** [comment]
Hi my name is Angelo and I recently got rx 6650 xt support with nootrx on mac os monterey installed on a mac pro 5.1. I tried to install Sonoma but with the boot arg I got stuck while loading the apple, now I have removed the boot arg and I'm trying to redo the Sonoma installation. Could there be support for Sonoma and kext integration on OCLP in the future?
